content/_code-samples/tx-serialization/js/binary_outputs.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
It doesn't make sense to me to commit an output file to the repository like this.

Reply:
removing this file